rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -2558,11 +2583,15 @@ void game_list_frame::PopulateGameList()
 
 	int row = 0;
 	int index = -1;
+
+	// Fallback is not needed when at least one entry is visible
+	const bool use_search_fallback = std::none_of(m_game_data.begin(), m_game_data.end(), [this](auto& game){ return IsEntryVisible(game); });
```

Comment:
this seems to be missing from PopulateGameGrid

Reply:
Fixed.